code/src/components/app-snackbar/AppSnackbar.tsx

```diff
@@ -0,0 +1,28 @@
+import Snackbar from "@mui/material/Snackbar";
+import Alert from "@mui/material/Alert";
+import useSnackbar from "@/hooks/use-snackbar/useSnackbar";
+
+import "@/components/app-snackbar/AppSnackbar.scss";
+
+export default function AppSnackbar() {
```

Comment:
Firstly we define const function and at the bottom use `export default`
Everything should be in one approach like our previous component with export

Reply:
Definitely. Fixed